users-service/service.yml

```diff
@@ -26,4 +26,7 @@ messages:
 
 
 dependencies:
-  - 'mongo'
+  mongo:
+    prod: 'ds143588.mlab.com:43588'
+    dev: 'mongo:27017'
+    test: 'localhost:27017'
```

Comment:
@trushton do tests use the mongo docker image as well? The code that I branched from used `localhost` in a testing environment